src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1,9 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Buffers;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
+using System.Runtime.CompilerServices;
```

Comment:
You may be able to get rid of both the _CompilerServices_ and the _InteropServices_ imports. Check other comments within this file for how to remove references to `MemoryMarshal` and `Unsafe`.

Reply:
Still used in GetHashCode to reinterpret `Span<uint>` as `Span<byte>`
Also used in `BitInteger.Multiply` to identify that both spans reference the same region of memory of the same length. In this case, multiplication can be effectively replaced by square operation.

Reply:
Unused namespace imports removed across several files.